kinbot/optimize.py

```diff
@@ -96,6 +97,7 @@ def do_optimization(self):
                     # do open chain part if cyclic part is done
                     if self.sconf == -1:
                         # open chain part has not started yet
+                        print(len(self.species.confs.cyc_conf_geoms))
```

Comment:
This print needs to go

Reply:
Sorry about missing some print statements. I'll make sure they are all cleaned up in the next commit.